src/main/java/ladder/domain/Lines.java

```diff
@@ -18,15 +19,29 @@ private void validLadderLength(int value) {
             throw new IllegalArgumentException("The ladder height should be larger than zero.");
     }
 
-    private List<Line> createLadder(int height, int width, LineGenerator generator) {
+    private List<Line> createLadder(int height, int width, RungGenerator generator) {
         return IntStream.range(0, height)
                 .mapToObj(i -> new Line(width, generator))
                 .collect(Collectors.toList());
```

Comment:
```suggestion
        return Stream.generate(() -> new Line(width, generator))
                .limit(height)
                .collect(Collectors.toList());
```
증감되는 인덱스를 사용할 필요가 없다면 
무한 스트림에서 리밋으로 생성하는 방법도 있으니 참고해 주세요